cmd/init.go

```diff
@@ -311,7 +311,7 @@ func install(config *initCommandConfig) error {
 
 	// reset config.StackRegistry and get it again from the newly untarred .appsody-config.yaml
 
-	defaultStackRegistry := getDefaultStackRegistry(config.RootCommandConfig)
+	//defaultStackRegistry := getDefaultStackRegistry(config.RootCommandConfig)
```

Comment:
This change would break the scenario in which Jane changes the `~/.appsody/.appsody.yaml` file and sets `images` to something other than `docker.io` (which is the default). If she sets it to `myregistry.io`, `appsody init` should use `myregistry.io/<namespace>/<image name>` (unless the stack fully qualifies the image name).

Reply:
if she wants that wouldn't she use `appsody init incubator/noidejs --image-registry myregistry.io`

Reply:
Wouldn't we always use the registry from `~/.appsody/.appsody.yaml` if `appsody_config.yaml` did not have registry specified?

Reply:
This is actually okay because `~/.appsody/.appsody.yaml` is read later on and performs the overwrite in `getProjectConfig()`